app/decorators/manageiq/providers/openstack/cloud_manager/template_decorator.rb

```diff
@@ -0,0 +1,9 @@
+class ManageIQ::Providers::Openstack::CloudManager::TemplateDecorator < MiqTemplateDecorator
+  def single_quad
```

Comment:
You don't need this line since you're inheriting the method from `MiqTemplateDecorator`.